### ISSUE TITLE
Form Monthly Test

### ISSUE THREAD

**Issue by mhabimana** [africanmathsinitiative/Climsoft#139]
@Patowhiz  The text boxes for this form need further adjustment (see below) what I am seeing:
![image](https://user-images.githubusercontent.com/10340170/41775262-ede9c108-762b-11e8-889f-234755d08d7b.png)

**Comment by Patowhiz** [comment]
@mhabimana this form is not done yet. We have an issue with entity framework which @isedwards and @dannyparsons are working on. Hopefully we may work on it once the issues are resolved in time.

**Comment by mhabimana** [comment]
@Patowhiz Ok,sounds great.

**Comment by mhabimana** [comment]
@Patowhiz : In Climsoft Verison 3.2,this is how the Form Monthly Data works; when you type the monthly value (under value text box and hit the enter key) the system populates automatically the number of days in the Month in the Period text box (see below):
![image](https://user-images.githubusercontent.com/10340170/42265590-a441c6f4-7f7c-11e8-82a6-9b48710d4c24.png)

**Comment by Patowhiz** [comment]
Should this happen (period populates) even when the value is indicated as missing?

**Comment by mhabimana** [comment]
@Patowhiz : Yes,because it is a missing for a  Month with a specific time period (see below): In this case,the missing character (M) and the Period inputs populates automatically once the enter key is pressed.
![image](https://user-images.githubusercontent.com/10340170/42266238-825d7996-7f7e-11e8-99cf-e47770c983ac.png)

**Comment by Patowhiz** [comment]
@mhabimana I just noticed from the screenshots above of climsoft v3, the months are in their corresponding words instead of numbers. Should we have the current numbers changed to their corresponding words or continue using the month numbers?

**Comment by mhabimana** [comment]
@Patowhiz : Thanks, yes,it would be great to have the labelling in words like in climsoft V3.x. Feel free to change this please.

**Comment by Patowhiz** [comment]
@mhabina. I'm implementing the **upload** functionality for this form. It doesn't have a day and an hour column. Yet we need to have a **date** object which will be stored to the `obsDatetime `of the `observationinitial `table. What is the recommended default day and hour. @smachua uses **1** as **day** value and **6** as the **hour** value, I thought this should be day=**last day of the month**.

**Comment by Patowhiz** [comment]
@mhabimana @isedwards @dannyparsons add new won't go to a new record for this form because it relies on table seq_year as it's sequencer, which is not available even in the updated model. I'll add this as separate issue

**Comment by Patowhiz** [comment]
This is how the new form monthly will look like 

![frmmonthly](https://user-images.githubusercontent.com/21179007/42638149-97bb9f7a-85f5-11e8-8a1b-fc3de254bcea.JPG)
